### PR TITLE
ACM-9569: add note about uniqueness of hosted cluster name

### PR DESCRIPTION
**Pull request by rokej**
Doc task: https://issues.redhat.com/browse/ACM-10129

Some limitations which cannot be prevented programmatically were discovered so these need to be documented.

### CODE REVIEW

**Comment by rokej** [comment]
@xenolinux Can you please review? Thank you.

File: clusters/hosted_control_planes/aws_intro.adoc [hunk 1]
```diff
@@ -7,11 +7,11 @@ A _hosted cluster_ is an {ocp-short} cluster with its API endpoint and control p
 
 *Important:*
 
-- Each hosted cluster must have a unique name in order for {mce-short} to manage it.
+- Each hosted cluster must have a unique name cluster-wide and cannot be the same as any existing managed cluster in order for {mce-short} to manage it.
```
Comment (xenolinux) [inline clusters/hosted_control_planes/aws_intro.adoc]:
```suggestion
- Each hosted cluster must have a cluster-wide unique name. A hosted cluster name cannot be the same as any existing managed cluster in order for {mce-short} to manage it.
```

File: clusters/hosted_control_planes/aws_intro.adoc [hunk 2]
```diff
@@ -7,11 +7,11 @@ A _hosted cluster_ is an {ocp-short} cluster with its API endpoint and control p
...
 
-- Run the hub cluster and workers on the same platform for hosted control planes.
+- Do not name a hosted cluster `clusters`.
```
Comment (xenolinux) [inline clusters/hosted_control_planes/aws_intro.adoc]:
```suggestion
- Do not use `clusters` as a hosted cluster name.
```

**Review by xenolinux — COMMENTED** [review]
2 suggestions that are applicable throughout the PR; otherwise LGTM!
Thanks!!

**Comment by swopebe** [comment]
@xenolinux and @rokej as peer review, I committed suggestions and changed the remaining files to match. Will approve and unless there is a need for more change, feel free to merge when ready.

**Review by swopebe — APPROVED** [review]
edited and approved

File: clusters/hosted_control_planes/managing_hosted_aws.adoc [hunk 3]
```diff
@@ -3,7 +3,7 @@
 
 You can use the hosted control plane command line interface, `hcp`, to create an {ocp-short} hosted cluster. The hosted cluster is automatically imported as a managed cluster. If you want to disable this automatic import feature, see xref:../hosted_control_planes/disable_auto_import.adoc#hosted-disable-auto-import[Disabling the automatic import of hosted clusters into {mce-short}].
 
-*Note:* Each hosted cluster must have a unique name in order for {mce-short} to manage it.
+*Note:* Each hosted cluster must have a unique name cluster-wide and cannot be the same as any existing managed cluster in order for {mce-short} to manage it.
```
Comment (xenolinux) [inline clusters/hosted_control_planes/managing_hosted_aws.adoc]:
```suggestion
*Note:* Each hosted cluster must have a cluster-wide unique name. A hosted cluster name cannot be the same as any existing managed cluster in order for {mce-short} to manage it.
```

Comment (xenolinux) [reply clusters/hosted_control_planes/managing_hosted_aws.adoc]: To keep the text consistent, this sentence can be changed to match with other files.

**Review by xenolinux — COMMENTED** [review]
One minor change; then we are good to merge.

**Comment by xenolinux** [comment]
@swopebe Thank you for updating the PR. @rokej Thanks for creating the PR.
I commited a [change](https://github.com/stolostron/rhacm-docs/pull/6166#discussion_r1512274131) in `managing_hosted_aws.adoc` which is same as the change in other files.
Merging since the PR is ready now.

**Pull request merged**